README.md

```diff
@@ -37,6 +41,10 @@ React Native date & time picker component for iOS and Android
     - [`maximumDate` (`optional`)](#maximumdate-optional)
     - [`minimumDate` (`optional`)](#minimumdate-optional)
     - [`timeZoneOffsetInMinutes` (`optional`, `iOS only`)](#timezoneoffsetinminutes-optional-ios-only)
+    - [`timeZoneOffsetInSeconds` (`optional`, `Windows only`)](#timezoneoffsetinsecond-optional-windows-only)
+    - [`dayOfWeekFormat` (`optional`, `Windows only`)](#dayOfWeekFormat-optional-windows-only)
+    - [`dateFormat` (`optional`, `Windows only`)](#dateFormat-optional-windows-only)
+    - [`firstDayOfWeek` (`optional`, `Windows only`)](#firstDayOfWeek-optional-windows-only)
```

Comment:
these 3 properties are part of locale data, do we really need separate properties that are Windows only?

Reply:
Filed #161